### PR TITLE
nav bar created and changes size at one breaking point

### PR DESCRIPTION
**Pull request by jstavh**
Created a nav bar that points homepage to careers page. It changes size at one breaking point that changes size for small screens (phones) to large screens (computers). This is higher than the jason-button branch

### CODE REVIEW

File: src/astro/src/components/Nav.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+                <a href="/careers" className="text-xl sm:text-2xl text-white no-underline">Careers</a>
+                <a href="mailto:hello@granola.team" className="text-xl sm:text-2xl text-white no-underline">Contact</a>
+            </nav>       
+        </div>
+    )
+}
+
+export default Nav; 
+
+// import * as React from 'react';
```
Comment (robinbb) [inline src/astro/src/components/Nav.jsx]: Remove commented out code.

**Review by robinbb — CHANGES_REQUESTED** [review]
Make PR against 'main' so that I can see a Netlify preview.

File: src/astro/src/components/Nav.jsx (hunk 1 above)
Comment (jstavh) [reply src/astro/src/components/Nav.jsx]: Removed comments.

**Comment by robinbb** [comment]
@jstavh Please modify Netlify settings so that *any* PR generates a preview, if possible.

**Comment by jstavh** [comment]
Netlify settings are set to preview every PR. I will need to continue to investigate why no preview is showing up here. I made these selections on Netlify via these instructions: Site settings > Build & deploy > Continuous Deployment > Deploy Previews.

**Comment by robinbb** [comment]
Netlify is probably confused because this PR was originally made against another base branch, and then converted into a PR against 'main'. You will have to re-create the PR, @jstavh .

**Pull request closed without merging**